CUSTOMISE_PORTFOLIO.md

```diff
@@ -0,0 +1,79 @@
+# Customize it to make your own portfolio ✏️
+
+In this project, there are basically 4 things that you need to change to customize this to anyone else's portfolio: **package.json**, **Personal Information**, **Github Information** and **Splash Logo**.
+
+### package.json
+
+Open this file, which is in the main cloned directory, choose any "name" and change "homepage " to `https://<your-github-username>.github.io`. Do not forget the `https://`, otherwise fonts will not load.
+
+### Personal Information
```

Comment:
Yes we should do this. But if we are doing this then we should add detailed description about each component in `portfolio.js`

Reply:
Ok in my free time 😬👍